src/internal/clusterstate/v2.7.0/util.go

```diff
@@ -0,0 +1,19 @@
+package v2_7_0
+
+import "github.com/pachyderm/pachyderm/v2/src/pfs"
```

Comment:
Should these just be methods (`.Key() string`) on the protos in src/pfs/pfs.go?

Reply:
Good idea! I think the original intent was that the `Key()` is more a db specific thing, so should be in `src/internal/pfsdb`. However, we can't import `pfsdb` into `clusterstate` because it introduces a circular dependency in Go imports. I don't mind adding these to `src/pfs/pfs.go` unless others are strongly opposed.